backend/users/forms.py

```diff
@@ -58,3 +63,57 @@ def save(self):
         user = authenticate(email=email, password=password)
         login(self.context, user)
         return user
+
+
+class UpdateForm(ContextAwareModelForm):
+    date_birth = CharField(required=False)
+    # TODO: Why if I tell you required=FALSE, the mutation fail if I don't put
+    # Bool! (instead of 'Bool')?
+    open_to_recruiting = BooleanField(required=False)
+    open_to_newsletter = BooleanField(required=False)
+    country = CharField(required=False)
+
+    def clean(self):
+
+        if not self.context.user.is_authenticated:
+            raise ValidationError(_("Must authenticate to update User information"))
+
+        return super().clean()
+
+    def save(self, commit=True):
+        super().save(commit=commit)
+        return User.objects.get(id=self.context.user.id)
```

Comment:
I think `self.instance` should already be up to date with the changes? Maybe we can avoid a new query to get the updated?

Reply:
Yup, of course!